### ISSUE TITLE
YawLoss seems not work in code

### ISSUE THREAD

**Issue by luoru** [MaverickPeter/vDiSCO#1]
yaw loss not work in trainer.py
"if yaw_loss is not None:
                        total_loss = loss
                    else:
                        total_loss = loss
"
fourier_spectrum not output in FusionNet
DeformableNet and DeformAttnFusionNet not included

Overall, it looks like the code is incomplete
^_^

**Comment by MaverickPeter** [comment]
@luoru Hi, thanks for following our work. This repo is under construction, and we will update it in the next version this month.
